src/test/java/entities/CardTest.java

```diff
@@ -0,0 +1,61 @@
+package entities;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class CardTest {
+
+    Card regCard;
```

Comment:
Add access modifiers to class attributes and methods

Reply:
Fixed for all test classes.